src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = sourceString
+    .split(';')
+    .map((style) => style.trim())
+    .reduce((acc, curr) => {
+      const [key, value] = curr.split(':').map((str) => str.trim());
```

Comment:
Consider adding a check to ensure that `curr.split(':')` results in exactly two elements. This will prevent potential errors if the input string is malformed or if there are missing colons.